### ISSUE TITLE
Memory Error

### ISSUE THREAD

**Issue by j07nikita** [facebookresearch/DrQA#190]
$ python3 scripts/pipeline/interactive.py 
12/08/2018 06:03:16 AM: [ Running on CPU only. ]
12/08/2018 06:03:16 AM: [ Initializing pipeline... ]
12/08/2018 06:03:16 AM: [ Initializing document ranker... ]
12/08/2018 06:03:16 AM: [ Loading /scratch/nikita/DrQA/data/wikipedia/docs-tfidf-ngram=2-hash=16777216-tokenizer=simple.npz ]
12/08/2018 06:03:48 AM: [ Initializing document reader... ]
12/08/2018 06:03:48 AM: [ Loading model /scratch/nikita/DrQA/data/reader/multitask.mdl ]
12/08/2018 06:03:52 AM: [ Initializing tokenizers and document retrievers... ]

Interactive DrQA
>> process(question, candidates=None, top_n=1, n_docs=5)
>> usage()

>>> salloc: Exceeded job memory limit

getting memory error 
using 20 cpu (20gb+)

**Comment by ajfisch** [comment]
Are you running on SLURM? I'm not to sure how the resource manager works there. Is overcommit enabled?

**Comment by j07nikita** [comment]
Yeah i am running on SLURM. How can i reduce memory usage (because using more threads can cause this issue) ?
